### PR TITLE
Add gizmo models/prefabs, rename SDK\Resources -> SDK\StandardAssets

### PR DESCRIPTION
**Pull request by davidkline-ms**
Overview
---
Add fbx model of a coordinate gizmo with; X increasing to the left, Y increasing upward, Z increasing away from the origin.

Add prefab for gizmo that scales and replaces the built-in materials with MRTK_Standard materials.

Rename Resources folder to StandardAssets as that is the true purpose and since Unity treats Resources as a special folder (all contents built into projects so that they can be runtime loaded) it is advisable to not use that name and allow developers to control the *true* Resources folder.

### CODE REVIEW

**Comment by SimonDarksideJ** [comment]
Definitely need the other hand for the Gizmo controller model to complete this PR.

I suspect we'll need subfolders in the prefabs to make finding things like controllers easier.  Especially when we have prefab for all the different SDK types.

In my visualizer PR, I add subfolders for each SDK type.

**Comment by davidkline-ms** [comment]
>  Especially when we have prefab for all the different SDK types.

If I read this correctly, the StandardAssets package will contain optional assets for ALL sdk packages

**Comment by StephenHodgson** [comment]
What do they look like?

**Comment by davidkline-ms** [comment]
I'll post an image of the right hand model this afternoon.

**Comment by StephenHodgson** [comment]
If it's just the arrows you posed in the Slack channel, why not just use the built in gizmos in the editor?

File: Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    400038: transform9
+    2100000: phong2
+    2100002: phong4
+    2100004: phong3
+    2300000: pCone6
+    3300000: pCone6
+    4300000: pCone6
+  externalObjects: {}
+  materials:
+    importMaterials: 1
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta]: let's turn off import materials and use the standard materials we've setup.

File: Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    isReadable: 1
+  meshes:
+    lODScreenPercentages: []
+    globalScale: 1
+    meshCompression: 0
+    addColliders: 0
+    importVisibility: 1
+    importBlendShapes: 1
+    importCameras: 1
+    importLights: 1
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta]: we probably don't wanna import camera's or lights either.

File: Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta]: I can do that

File: Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta (hunk 2 above)
Comment (davidkline-ms) [reply Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta]: sounds good

**Comment by davidkline-ms** [comment]
> If it's just the arrows you posed in the Slack channel, why not just use the built in gizmos in the editor?

These were requested so that they could be used as "Controller" models.

**Comment by davidkline-ms** [comment]
Left gizmo added. @StephenHodgson's meta file changes made. Ready to review. Thx

**Comment by davidkline-ms** [comment]
>Would still prefer the Gizmo controller in a sub folder for better organisation, but that can be organised later when I add the other controller models.

That's what I figured :)

**Pull request merged**